functionary/ui/templates/registration/login.html

```diff
@@ -1,34 +1,29 @@
-{% extends "base.html" %}
-
+{% extends "base_old.html" %}
```

Comment:
Can we call this `base_bulma.html`?  Also, inverting it and creating a `base_bootstrap.html` and then pointing pages at that as you convert them would have made for a smaller initial change set here, since pages that you're not intending to change would be untouched for now...

Reply:
I don't feel that strongly about it.  You can just do the rename to `base_bulma.html` and then carry on as you were.